src/_tables.scss

```diff
@@ -0,0 +1,34 @@
+//Mixin for table styles
+@use 'sass:color';
+
+@mixin table-styles {
```

Comment:
Same here with this mixin. Please move it into `_mixins.scss`, then push your changes. Thank you!

Reply:
Perfect, changes done and pushed.